instrumentation/java-http-client/javaagent/src/test/groovy/JdkHttpClientTest.groovy

```diff
@@ -64,40 +62,4 @@ class JdkHttpClientTest extends HttpClientTest<HttpRequest> implements AgentTest
   boolean testErrorWithCallback() {
     return false
   }
-
-  @Requires({ !System.getProperty("java.vm.name").contains("IBM J9 VM") })
-  def "test https request"() {
-    given:
-    def uri = new URI("https://www.google.com/")
-
-    when:
-    def responseCode = doRequest(method, uri)
-
-    then:
-    responseCode == 200
-    assertTraces(1) {
-      trace(0, 1) {
-        span(0) {
-          hasNoParent()
-          name expectedOperationName(method)
-          kind CLIENT
-          attributes {
-            "${SemanticAttributes.NET_TRANSPORT.key}" IP_TCP
-            "${SemanticAttributes.NET_PEER_NAME.key}" uri.host
-            "${SemanticAttributes.NET_PEER_IP.key}" { it == null || it == "127.0.0.1" }
-            // Optional
-            "${SemanticAttributes.NET_PEER_PORT.key}" uri.port > 0 ? uri.port : { it == null || it == 443 }
-            "${SemanticAttributes.HTTP_URL.key}" { it == "${uri}" || it == "${removeFragment(uri)}" }
-            "${SemanticAttributes.HTTP_METHOD.key}" method
-            "${SemanticAttributes.HTTP_FLAVOR.key}" "2.0"
```

Comment:
This test was only here to override the base class for this assertion. I forced the client to HTTP1 instead and will add proper HTTP2 tests in the future.

Reply:
Sure filed #3227